libs/ui-lib/lib/common/config/docs_links.ts

```diff
@@ -100,3 +100,7 @@ export const FEEDBACK_FORM_LINK =
 
 export const CHANGE_ISO_PASSWORD_FILE_LINK =
   'https://raw.githubusercontent.com/openshift/assisted-service/master/docs/change-iso-password.sh';
+
+//Ciso Intersight Link
```

Comment:
I don't think this comment is necessary (the function name is clear enough) but if we have to keep it, let's please fix the typo.